scripts/run_test_in_devicefarm.sh

```diff
@@ -83,6 +83,26 @@ if [[ -z "${minDevice}" || -z "${middleDevice}" || -z "${latestDevice}" ]]; then
     exit 1
 fi
 
+# Function to cancel duplicate runs for same code source in device farm
+# Function to cancel duplicate runs for same code source in device farm.
+function stopDuplicates {
+  echo "Stopping duplicate runs"
+  name="$file_name-$CODEBUILD_SOURCE_VERSION"
```

Comment:
I'm looking at the Codebuild docs here and it says "For GitHub... [`$CODEBUILD_SOURCE_VERSION`] is the commit ID, branch name, or tag name associated with the version" but also that "if it's triggered by a webhook pull request event it's `pr/pull-request-number`" which seems a little inconsistent.  Can you just confirm that this isn't going to change if we add a commit?

Reply:
That is basically the idea. If you publish another commit in the same PR (say pr/xyz), it will cancel the old tests running/pending for it.

Reply:
Yep, makes sense.  I guess I was just wondering why it listed the comment ID as another value that `$CODEBUILD_SOURCE_VERSION` can be -- if it's not a part of a PR webhook does it use that value?  Anyways if your `commit to test` worked then I'm good!